Monika After Story/game/script-holidays.rpy

```diff
@@ -3590,6 +3593,8 @@ label mas_player_bday_cake:
         else:
             m 1ekbfa "I love you, [player]!"
     $ mas_rmallEVL("mas_player_bday_no_restart")
+    # "love" return key won't work here without adding a bunch of return _returns, so we'll set this manually
+    $ persistent._mas_last_monika_ily = datetime.datetime.now()
```

Comment:
to keep things consistent we should return "love" here too and add appropriate returns where necessary.

Reply:
actually, rather than what I said, lets just have a quick function like `mas_ILY` or something that will set the appropriate persistent to a given datetime, defaulting to now. This should be used in all cases where a return "love" would have to be passed thru multiple returns. (or for the idle mode case)
Additionally, all return keys should only be used by events called with `call_next_event` so that contributors wont assume that every `return` will respect a return key.

Reply:
Why create a function to set a single var instead of just setting the var?